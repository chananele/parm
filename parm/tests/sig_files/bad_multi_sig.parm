---
name: bad_multi_p1
exports:
  - exp1
method: match
pattern: |
  % goto(0x103E8)
  ... {3}
  exp1: BX LR
---
name: bad_multi_p2
imports:
  - exp1
exports:
  - exp2
method: match
pattern: |
  % goto("exp1")
  ... {2}
  exp2:
...
