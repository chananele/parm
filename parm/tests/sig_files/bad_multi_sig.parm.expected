name: bad_multi_p1
result: failure
errors:
  - symbol [exp1] was not exported
---
name: bad_multi_p2
result: not run
errors:
  - unresolved import [exp1]
